src/components/EditGrower.js

```diff
@@ -158,21 +158,25 @@ const EditGrower = (props) => {
           {inputs.map((row, rowIdx) => (
             <Grid item container direction="row" key={rowIdx}>
               {row.map((input, colIdx) => (
-                <TextField
-                  key={`${rowIdx}_${colIdx}`}
-                  className={classes.textInput}
-                  id={input.attr}
-                  label={input.label}
-                  type={input.type || 'text'}
-                  variant="outlined"
-                  InputLabelProps={{
-                    shrink: true,
-                  }}
-                  onChange={(e) => {
-                    handleChange(input.attr, e.target.value);
-                  }}
-                  value={getValue(input.attr)}
-                />
+                <>
+                  {!(input.type === 'email') && !(input.type === 'tel') && (
```

Comment:
Rather than conditionally omitting inputs, it would be best to remove these items from the `inputs` array altogether. These values are not used anywhere else.

Reply:
@nmcharlton done . Look into this solution.